projects/canopy/src/lib/forms/sort-code/sort-code.notes.ts

```diff
@@ -31,6 +31,7 @@ and in your HTML:
 ### LgSortCodeComponent
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
+| \`\`invalidSortCode$\`\` Subject that is passed from the parent form if the sort code fields are empty in order to mark the fields and set an error so that the validity state gets propagated | Subject<void> | No |
```

Comment:
Thanks for adding this. I'm still unsure about how to use this in an application, so perhaps we should have a section in the notes explaining how to use it within a form. 

E.g.
<img width="675" alt="Screenshot 2021-07-28 at 09 49 26" src="https://user-images.githubusercontent.com/1943532/127293369-c4b21202-ed98-46ed-aeff-0de8a112585d.png">

Reply:
We've discussed the solution with @Zimann and we are trying to investigate a way not to have this input.